### PR TITLE
[MeshingApplication][CMake] Add `mmg/common` when looking for header

### LINKED ISSUE #10560: [MeshingApplication] Compilation issue
**Description**
Dear all,

I just tried to compile the `MeshingApp` in windows (in linux is the same) and I got the following compilation errors:

~~~
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(643): error C3861: 'MMG2D_G
et_quadrilateral': no se encontró el identificador [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication\
KratosMeshingCore.vcxproj]
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(1408): error C3861: 'MMG2D_
Get_quadrilateral': no se encontró el identificador [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication
\KratosMeshingCore.vcxproj]
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(1695): error C2065: 'MMGS_I
PARAM_optim': identificador no declarado [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication\KratosMesh
ingCore.vcxproj]
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(1709): error C2660: 'MMG2D_
Set_meshSize': la función no acepta 5 argumentos [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication\Kr
atosMeshingCore.vcxproj]
~~~

among many more...

Should I update anything?

Regards,

@JoseMGonzalez-UPC 

I attach my configure.bat:

~~~ bat
rem Please do not modify this script

rem For any question please contact with us in:
rem  - https://github.com/KratosMultiphysics/Kratos

rem Optional parameters:
rem You can find a list will all the compiation options in INSTALL.md or here:
rem  - https://github.com/KratosMultiphysics/Kratos/wiki/Compilation-options

call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat"

rem Set compiler
@echo off
set CC=cl.exe
set CXX=cl.exe

rem Set variables
set KRATOS_SOURCE=~0,-1%/..
set KRATOS_BUILD=%KRATOS_SOURCE%/build
set KRATOS_APP_DIR=applications
set KRATOS_INSTALL_PYTHON_USING_LINKS=ON

rem Set basic configuration
set KRATOS_BUILD_TYPE=Release
set BOOST_ROOT=D:\boost_1_69_0
set PYTHON_EXECUTABLE=C:\Program Files\Python310\python.exe

rem Set applications to compile
set KRATOS_APPLICATIONS=
CALL :add_app %KRATOS_APP_DIR%\LinearSolversApplication;
CALL :add_app %KRATOS_APP_DIR%\StructuralMechanicsApplication;
CALL :add_app %KRATOS_APP_DIR%\ConstitutiveLawsApplication;
rem CALL :add_app %KRATOS_APP_DIR%\ContactStructuralMechanicsApplication;
CALL :add_app %KRATOS_APP_DIR%\FemToDemApplication;
CALL :add_app %KRATOS_APP_DIR%\PfemFluidDynamicsApplication;
CALL :add_app %KRATOS_APP_DIR%\DelaunayMeshingApplication;
CALL :add_app %KRATOS_APP_DIR%\DEMApplication;
CALL :add_app %KRATOS_APP_DIR%\MeshingApplication;
CALL :add_app %KRATOS_APP_DIR%\DemStructuresCouplingApplication;

rem Clean
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\cmake_install.cmake"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeCache.txt"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"

rem Configure
@echo on
cmake -G"Visual Studio 15 2017 Win64" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"  ^
-DINCLUDE_FEAST=OFF                                                      ^
-DINCLUDE_MMG=ON                                                         ^
-DUSE_TRIANGLE_NONFREE_TPL=ON                                            ^
-DMMG_BUILDDIR="D:/mmg_master/buid"                                      ^
-DMMG_LIBDIR="D:/mmg_master/buid/lib/Release"                            ^
-DCMAKE_CXX_FLAGS=" -D_SCL_SECURE_NO_WARNINGS /MP5"                      ^
-DUSE_EIGEN_MKL=ON                                                       ^
-DINSTALL_RUNKRATOS=OFF                                                  ^
-DUSE_COTIRE=OFF                                                         ^
-DKRATOS_BUILD_TESTING=ON

rem Build
cmake --build "%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%" --target install -- /property:configuration=%KRATOS_BUILD_TYPE% /p:Platform=x64
goto:eof

rem Function to add apps
:add_app
set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%1;
goto:eof

~~~

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Add mmg/common when looking for header. Fixes #10560

**🆕 Changelog**

- [Add mmg/common when looking for header](https://github.com/KratosMultiphysics/Kratos/commit/775b5cc141b9e255b803f9cc6aa85cb1d7cc6d87)

### CODE REVIEW

File: cmake_modules/FindMMG2D.cmake
Comment (marcnunezc) [inline cmake_modules/FindMMG2D.cmake]:
```suggestion
  if(MMG_DIR)
```

Unwanted change

**Comment by marcnunezc** [comment]
Thanks! Lets wait for the OK from mmg before merging this.

**Comment by AlejandroCornejo** [comment]
Thank you guys, so to summarize, now what should we put in the configure??

**Comment by AlejandroCornejo** [comment]
> Thank you guys, so to summarize, now what should we put in the configure??

I've tried and now seems to compile properly @JoseMGonzalez-UPC

**Comment by loumalouomega** [comment]
@marcnunezc

**Review by marcnunezc — APPROVED** [review]
Ok I can approve but maybe we will need to update again this file if MMG suggests new changes.

In the meanwhile Im fine fixing it on our side.

**Pull request merged**
